docs/sql/statements/drop-constraint.rst

```diff
@@ -0,0 +1,33 @@
+.. _drop-constraint:
+
+===================
+``DROP CONSTRAINT``
+===================
+
+Remove a :ref:`check_constraint` constraont from the table.
+
+.. rubric:: Table of contents
+
+.. contents::
+   :local:
+
+Synopsis
+========
+
+.. code-block:: sql
+
+    ALTER TABLE table_ident DROP CONSTRAINT check_ident
```

Comment:
```suggestion
    ALTER TABLE table_ident DROP CONSTRAINT check_name
```

To match the synposis definition used elsewhere `constraint <name> check(boolean_expression)`